Spatial_searching/include/CGAL/Kd_tree_node.h

```diff
@@ -51,15 +51,12 @@ namespace CGAL {
     typedef typename Kdt::iterator iterator;
     typedef typename Kdt::D D;
 
-    bool leaf;
```

Comment:
is that change required in this PR ?

Reply:
No.

Reply:
it is not clear to me that this will not negatively impact the performance so except if you show that it is faster you should undo that change.